### DISCUSSION TITLE
Small aesthetic bug in figma anchor component

### DISCUSSION THREAD

**Discussion by frankfzhao** [twilio-labs/paste#2798] [Help: Figma] [answered]
Notice a small aesthetic bug in figma anchor component.
See screenshot below, if icon is enabled for anchor,  the icon and the text are not center aligned.

<img width="513" alt="Screenshot 2022-11-02 at 12 46 40 PM" src="https://user-images.githubusercontent.com/82524550/199576176-6b99bdce-2ce9-4832-ba04-510abf817276.png">

**Comment by SiTaggart** [comment]
Hi @frankfzhao 

Thanks for spotting that. As it's so small, might this be something that you can consider fixing and contributing back to the system?

Thanks

**Comment by ghost** [comment]
@frankfzhao just a bump here to see if you were able to take a look at this? If not let us know and we can file a low priority issue to get this resolved in the future!

**Comment by TheSisb** [comment, marked as answer]
Closing because stale.

Aside: you may want to double check if it's a bug in your file? The screenshot shows the Anchor is missing an underline...  You can also try to run `cmd+P` > `Repair component connections` and see if that fixes it. Or try to bring a new instance of Anchor in your file.
